.circleci/config.yml

```diff
@@ -2,10 +2,20 @@ version: 2.1
 
 workflows:
   version: 2.1
-  test:
+  test-and-deploy:
     jobs:
       - unit_test
       - build_and_test_docker
+      # TODO: add approval step to control when deploys take place
```

Comment:
Rather than do this via circleci, I think it'd be nicer to have this step publish a new application version to EB, but not deploy it. Then through the UI at https://console.aws.amazon.com/elasticbeanstalk/home?region=us-east-1#/application/versions?applicationName=covid-publishing-api one of us can go in and hit deploy.

One advantage of that is you can create the new version, push it to a staging env, wait for a while, then push it to a prod env.

Also it's how I've done it before :)

Reply:
Weirdly I can't figure out if that's possible through the `eb` CLI. All I can find is the more general AWS CLI's command to do it: https://awscli.amazonaws.com/v2/documentation/api/latest/reference/elasticbeanstalk/create-application-version.html

Reply:
lol because of course they've like, half-heartedly deprecated the eb CLI: "Previously, Elastic Beanstalk supported a separate CLI that provided direct access to API operations called the Elastic Beanstalk API CLI. This has been replaced with the AWS CLI, which provides the same functionality but for all AWS services' APIs." from https://docs.amazonaws.cn/en_us/elasticbeanstalk/latest/dg/eb-cli3.html

I'll take a crack at using that approach this weekend when I get some time.

Reply:
How deeply strange of them to kinda sorta have both CLIs. Agreed that doing the environment swap thing makes sense in the future once we have a production environment worth protecting from untested changes.